### ISSUE TITLE
[Achievement: "Mining Your Own Business: La Noscea IV] not working, stands still in first pos

### ISSUE THREAD

**Issue by HermitBiscuit** [awgil/ffxiv_visland#81]
Hello,

Tried the first one, which was the achievement  for mining in La Noscea III and that worked fine (with some minr hiccup). I then tried the one for IV and it simply doesn't want to go anywhere. It stands still in one spot, the first coordinates. I've also tried removing the first line (Position) to see if it works after the second one but no, same there. It moves to the new first and stops there. Doesn't continue.

I went to try Black Shroud III and that works fine as well so of the three I've tried, only one is misbehaving. All three have been handled the exact same way with copy, import and play. No manual inputs during the first few runs just to make sure things work.

**Comment by HermitBiscuit** [comment]
Well, since I don't just wanna complain but also try my hand at fixing an issue, I made my own route that I've ran for a solid 30min without issues. It's not perfect, but it gets the job done. Feel free to use it if the current one is indeed faulty for other people as well (again, only La Noscea IV was faulty the rest worked fine so far).

`{"Name":"La Noscea IV","Group":"Ungrouped","Food":0,"Manual":0,"TargetGatherItem":0,"Waypoints":[{"Position":{"X":309.71695,"Y":44.47461,"Z":311.9359},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":322.3453,"Y":44.24784,"Z":309.4849},"ZoneID":137,"Radius":2.0,"Movement":0,"Pathfind":false,"InteractWithOID":30501,"InteractWithName":"mineral deposit","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":0,"IsNode":true,"NodeJob":16},{"Position":{"X":330.18677,"Y":40.791153,"Z":313.13168},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":338.7641,"Y":38.927402,"Z":311.02533},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":370.05743,"Y":30.430761,"Z":310.79602},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":371.80966,"Y":31.47505,"Z":289.50693},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":373.46115,"Y":30.474531,"Z":308.53842},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":383.27737,"Y":31.28987,"Z":299.36557},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":397.39352,"Y":28.454117,"Z":290.154},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":385.8738,"Y":31.984468,"Z":286.81064},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":387.2165,"Y":32.57661,"Z":289.1993},"ZoneID":137,"Radius":2.0,"Movement":0,"Pathfind":false,"InteractWithOID":30500,"InteractWithName":"mineral deposit","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":0,"IsNode":true,"NodeJob":16},{"Position":{"X":431.01535,"Y":18.361998,"Z":303.79398},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":437.15445,"Y":17.023203,"Z":292.69324},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":447.92276,"Y":15.1350155,"Z":282.2999},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":446.77026,"Y":15.807781,"Z":271.12512},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":445.43396,"Y":16.417973,"Z":255.30428},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":440.5469,"Y":15.337154,"Z":241.0444},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":436.1784,"Y":16.951529,"Z":248.43903},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":425.7838,"Y":22.396692,"Z":255.57634},"ZoneID":137,"Radius":3.0,"Movement":0,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":425.5067,"Y":22.16839,"Z":256.7983},"ZoneID":137,"Radius":2.0,"Movement":0,"Pathfind":false,"InteractWithOID":30499,"InteractWithName":"mineral deposit","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":0,"IsNode":true,"NodeJob":16},{"Position":{"X":385.75882,"Y":30.003971,"Z":309.46396},"ZoneID":137,"Radius":3.0,"Movement":2,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0},{"Position":{"X":335.62372,"Y":39.061195,"Z":321.08112},"ZoneID":137,"Radius":3.0,"Movement":2,"Pathfind":true,"InteractWithOID":0,"InteractWithName":"","InteractWithPosition":{"X":0.0,"Y":0.0,"Z":0.0},"showInteractions":false,"Interaction":1,"EmoteID":0,"ItemID":0,"ActionID":0,"QuestID":0,"QuestSeq":0,"MobID":0,"StopCondition":0,"KillCount":0,"RouteName":"","ChatCommand":"","showWaits":false,"WaitForCondition":0,"WaitTimeMs":0,"WaitTimeET":{"X":0.0,"Y":0.0},"IsPhantom":false,"Path":null,"GatheringType":99,"IsNode":false,"NodeJob":0}]}`
